src/modules/coach/containers/onboarding-questionnaire/utils/validation.ts

```
@@ -0,0 +1,119 @@
+      default: {
+        shape[id] = Yup.mixed().nullable().optional();
+      }
+    }
+
+    if (conditional_display) {
+      const { depends_on, operator, value: condValue } = conditional_display;
+
+      shape[id] = (shape[id] as Yup.Schema).when(depends_on, {
+        is: (depVal: unknown) =>
+          evaluateConditionalDisplay(operator, depVal, condValue),
+        then: (schema: Yup.Schema) =>
```

Biome (2.4.6)

[error] Do not add then to an object.

(lint/suspicious/noThenProperty)